CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2025-01-06
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as `v20250106`.
+
+### Announcements
+* AKS Kubernetes version 1.31 is now in GA.
+* 
+### Release Notes
+* Features:
+  * 
+* Preview features:
+
+* Behavior change:
+
```

Comment:
If these sections are empty, we should remove them. but I suggest checking internally to see if there were any changes introduced (if it wasn't mentioned in the PRs)